docs/source/developer/accessibility.rst

```diff
@@ -0,0 +1,172 @@
+.. Copyright (c) Jupyter Development Team.
+.. Distributed under the terms of the Modified BSD License.
+
+JupyterLab Accessibility: Developer's Guide
+===========================================
+
+If you're making changes to the JupyterLab source code and you're concerned
+about `accessibility <https://en.wikipedia.org/wiki/Accessibility>`__, this page
+is for you.
+
+Looking for other ways to `contribute to accessibility
+<https://jupyter-accessibility.readthedocs.io/en/latest/contribute/guide.html>`__?
+
+Where to start
+--------------
+
+A common question when accessibility-minded developers come to JupyterLab is:
```

Comment:
I like saying thank you, though I will admit the step above [Contribute page](https://jupyterlab.readthedocs.io/en/latest/developer/contributing.html) does as well.

```suggestion
Thank you for being interested in improving JupyterLab's accessibility. Whether you are making accessibility-specific fixes or considering the accessibility impacts of another contribution, the work you do betters JupyterLab for everyone who uses it. 

Once they begin exploring the space, a common question when accessibility-minded developers come to JupyterLab is:
```